### DISCUSSION TITLE
[@types/express-serve-static-core] 5.0.2 breaking change

### DISCUSSION THREAD

**Discussion by derheinz57** [DefinitelyTyped/DefinitelyTyped#71327] [Issues with a @types package]
An application I own suddenly stopped building last week, and I delved into it to find it's due to @types/express-serve-static-core:5.0.2. Version 5.0.1 works fine. Relevant PR that introduced the problem is here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71024

Wouldn't removing methods like this be considered a breaking change and warrant a new major version? Even though the removed functions had been deprecated, it seems other things dependent on it may still be using them. 

@xfournet @rbuckton

[a comment by an automated account is not shown]

**Comment by xfournet** [comment]
Theses methods have been removed from express 5 cf https://expressjs.com/en/guide/migrating-5.html). They were still present in @types/express-serve-static-core 5.0.0/5.0.1 but it was an oversight. With 5.0.0/5.0.1 it will not comply during build/check, but it will break at runtime.
In case you are still using express 4, you shouldn't update to @types/express-serve-static-core v5 and stick with @types/express-serve-static-core v4
